dashboard/src/app/navbar/navbar-dropdown-menu/navbar-dropdown-menu.controller.js

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2015-2016 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ */
+'use strict';
+
+
+/**
+ * This class is handling the controller for the dropdown menu on navbar
+ * @author Oleksii Kurinnyi
+ */
+export class NavbarDropdownMenuCtrl {
```

Comment:
+1 But it should be a separate commit(maybe, separate PR) to all of inclusions like  xxxxCtrl.